distribution.pm

```diff
@@ -54,6 +54,7 @@ sub add_console {
         'tty-console'     => 'ttyConsole',
         'ssh-xterm'       => 'sshXtermVt',
         'ssh-virtsh'      => 'sshVirtsh',
+        'ssh-virtsh-sut'  => 'sshVirtshSUT',
```

Comment:
I am not sure about the console class name. Currently we use `sut` console via `ssh-virtsh`, and in other backends, and it is about VNC, this one is not.

Reply:
To my understanding `ssh-virtsh` **is** SSH and not VNC. The SSH connection is used to configure VNC among other things, though. So I thought it would make sense to call it `ssh-virtsh-sut` because it is a SSH connection using `virsh console` to provide console access to the SUT.

But I'm not familiar with your conventions. So just tell me how to call it and I rename it (if it make some sort of sense).

Reply:
maybe `ssh-virtsh-serial`?

Reply:
I like this one.

Reply:
Ok, I'll change it to `ssh-virsh-serial`.